app/views/questions/show.html.haml

```diff
@@ -11,3 +11,14 @@
   %p
     %strong Body:
     = @question.body
+
+%h2 Answers
+
+= render 'answers'
+
+- if user_signed_in?
+  = form_for [@question, @answer], remote: true do |f|
+    = f.label :body, "Your answer"
+    .answers-errors
+    = f.text_field :body
+    = f.submit "Answer"
```

Comment:
Add a new line
(It would be better if you allow your IDE to do it automatically)